src/NuGet.Core/NuGet.ProjectModel/DependencyGraphSpec.cs

```diff
@@ -6,6 +6,8 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using System.Text;
+using System.Text.Json;
 using Newtonsoft.Json;
```

Comment:
Shouldn't this using no longer be necessary?

Reply:
write is still using nj.